terraform/terraform-init.sh

```diff
@@ -93,146 +117,126 @@ azure_location_to_shortloc () {
         echo "[+] Unsupported location '$LOCATION'"
         exit 1
     fi
-    echo "[+] Using location short name '$SHORTLOC'"
+}
+
+set_env () {
+    # Assign variables STATE_RG, STATE_ACCOUNT and PERSISTENT_RG: these
+    # variables are used to select the remote state storage and persistent
+    # data used in this ghaf-infra instance.
+    STATE_RG="ghaf-infra-0-state-${SHORTLOC}"
+    STATE_ACCOUNT="ghafinfra0state${SHORTLOC}"
+    PERSISTENT_RG="ghaf-infra-0-persistent-${SHORTLOC}"
+    echo "[+] Using state '$STATE_RG'"
+    echo "[+] Using persistent '$PERSISTENT_RG'"
+    echo "storage_account_rg_name=$STATE_RG" >"$MYDIR/.env"
+    echo "storage_account_name=$STATE_ACCOUNT" >>"$MYDIR/.env"
+    echo "persistent_rg_name=$PERSISTENT_RG" >>"$MYDIR/.env"
 }
 
 init_state_storage () {
```

Comment:
This can be misinterpreted as only doing `terraform init`, while it's actually also doing an apply.

Consider renaming this (and the echo) to something more descriptive, like `apply_state_storage`

Reply:
This can be done in a followup, shouldn't block this from landing!